ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -464,7 +464,18 @@ func addRegistryCredsToContainerDefs(containerDefs []*ecs.ContainerDefinition, c
 					CredentialsParameter: aws.String(foundCredParam),
 				}
 				containerDef.RepositoryCredentials = &containerRepoCreds
+
+				// remove container entry from cred map
+				delete(credsMap, containerName)
+			}
+		}
+		// if credMap contains unused containers, log a warning
```

Comment:
By unused containers, I assume you mean containers that don't exist in the docker compose file -- that could perhaps be more explicit in the comment and log message.